<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Presensi Bumijonet</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icons/F:/IMAGES/ICONS/Button-Add-blue-48.png</normaloff>:/icons/F:/IMAGES/ICONS/Button-Add-blue-48.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="1">
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>12</height>
          </size>
         </property>
         <property name="text">
          <string>OP 1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>136</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="8" column="0" colspan="2">
           <widget class="QLabel" name="labelKeterangan1">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>14</height>
             </size>
            </property>
            <property name="text">
             <string>keterangan: &lt;i&gt;(opsional)&lt;/i&gt;</string>
            </property>
            <property name="buddy">
             <cstring>keterangan1</cstring>
            </property>
           </widget>
          </item>
          <item row="10" column="0" colspan="2">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="sudahLogin1">
            <property name="styleSheet">
             <string notr="true">text-align: center;</string>
            </property>
            <property name="text">
             <string>login</string>
            </property>
           </widget>
          </item>
          <item row="11" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton1">
            <property name="text">
             <string>Login</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelUsername1">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>14</height>
             </size>
            </property>
            <property name="text">
             <string>username:</string>
            </property>
            <property name="buddy">
             <cstring>username1</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QLineEdit" name="username1"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="labelPassword1">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>14</height>
             </size>
            </property>
            <property name="text">
             <string>password:</string>
            </property>
            <property name="buddy">
             <cstring>password1</cstring>
            </property>
           </widget>
          </item>
          <item row="7" column="0" colspan="2">
           <widget class="QLineEdit" name="password1">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="9" column="0" colspan="2">
           <widget class="QLineEdit" name="keterangan1">
            <property name="styleSheet">
             <string notr="true">background-color: #f6f6f6;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>136</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>12</height>
          </size>
         </property>
         <property name="text">
          <string>OP 2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="0">
           <widget class="QLineEdit" name="username2"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="labelPassword2">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>14</height>
             </size>
            </property>
            <property name="text">
             <string>password:</string>
            </property>
            <property name="buddy">
             <cstring>password2</cstring>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLineEdit" name="password2">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelUsername2">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>14</height>
             </size>
            </property>
            <property name="text">
             <string>username:</string>
            </property>
            <property name="buddy">
             <cstring>username2</cstring>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QPushButton" name="pushButton2">
            <property name="text">
             <string>Login</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="sudahLogin2">
            <property name="styleSheet">
             <string notr="true">text-align: center</string>
            </property>
            <property name="text">
             <string>login</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="labelKeterangan2">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>14</height>
             </size>
            </property>
            <property name="text">
             <string>keterangan: &lt;i&gt;(opsional)&lt;/i&gt;</string>
            </property>
            <property name="buddy">
             <cstring>keterangan2</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="8" column="0">
           <widget class="QLineEdit" name="keterangan2">
            <property name="styleSheet">
             <string notr="true">background-color: #f6f6f6;</string>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QFrame" name="frame_3">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>45</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="loadingText">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>copyright © yudha 2013</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>username2</tabstop>
  <tabstop>password2</tabstop>
  <tabstop>keterangan2</tabstop>
  <tabstop>pushButton2</tabstop>
  <tabstop>username1</tabstop>
  <tabstop>password1</tabstop>
  <tabstop>keterangan1</tabstop>
  <tabstop>pushButton1</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
